File: 20140601/fizzbuzz.py
def toStr(num):
    if (num % 15 == 0):
        return "FizzBuzz"
    elif (num % 3 == 0):
        return "Fizz"
    elif (num % 5 == 0):
        return "Buzz"
    else:
        return num

def fizzBuzz(min,max):
    for i in range(min,max+1):
        print(toStr(i))

fizzBuzz(1, 100)
    

File: 20140712/fizzbuzz.js
function toStr(num){
  if (num % 15 == 0)
    return "FizzBuzz";
  else if (num % 3 == 0)
    return "Fizz";
  else if (num % 5 == 0)
    return "Buzz";
  else
    return num;
}

function fizzBuzz(min, max){
  for (var i = min; i <= max; i++) {
    print(toStr(i));
  }
}

fizzBuzz(1,100);